astroid/brain/brain_argparse.py

```diff
@@ -20,13 +20,10 @@ def infer_namespace(node, context: InferenceContext | None = None):
         "Namespace",
         lineno=node.lineno,
         col_offset=node.col_offset,
-        parent=nodes.Unknown(),
+        parent=AstroidManager().adhoc_module,  # this class is not real
```

Comment:
Not sure about the name of `adhoc`. But I also don't have better suggestions

Reply:
I can come up with other options. Please, tell what you are unsure about it, to guide the thought process for new names.

Reply:
@Pierre-Sassoulas Any suggestions here?

Reply:
Maybe ``synthetic_root_node``, ``placeholder_root_node``,  ``placeholder_parent_node``,  ``placeholder_module_node`` ?  wasn't sure what adhoc meant exactly before checking the definition to be honest.